src/components/organisms/NotificationsManager/Notification.tsx

```diff
@@ -60,7 +74,7 @@ const Notification: React.FC<NotificationProps> = props => {
           <S.CopyOutlined onClick={onCopyToClipboard} />
         </Tooltip>
         <S.MessageBodyContainer>
-          <S.TitleSpan>{title}</S.TitleSpan>
+          <S.TitleSpan>{modify(title)}</S.TitleSpan>
           <S.MessageSpan>
             <NotificationMarkdown notification={notification} type={notificationType} />
```

Comment:
Now that I looked again at the code, I think the method you created will have to be called somewhere in the `NotificationMarkdown` component, on the `notification.message` property, not on the `notification.title`

Reply:
so should i remove the changes from the `Notification` and `MessageBox` component?